arex-storage-web-api/src/main/java/com/arextest/storage/beans/StorageAutoConfiguration.java

```diff
@@ -46,7 +57,18 @@ public StorageAutoConfiguration(StorageConfigurationProperties configurationProp
     @Bean
     @ConditionalOnMissingBean(MongoDatabase.class)
     public MongoDatabase mongoDatabase() {
-        return MongoDbUtils.create(properties.getMongodbUri());
+        MongoDatabase database = MongoDbUtils.create(properties.getMongodbUri());
+        for (MockCategoryType category : MockCategoryType.DEFAULTS) {
+            setTTLIndex(category, database);
+        }
+        return database;
+    }
+
+    private void setTTLIndex(MockCategoryType category, MongoDatabase mongoDatabase) {
```

Comment:
If you want to wrap the behavior of adding a ttl index to a mocker-related collection into a method, the method needs to be more directional

Reply:
done